vector/build.gradle

```diff
@@ -110,6 +112,20 @@ android {
             resValue "string", "allow_ga_use", "false"
             resValue "string", "short_flavor_description", "F"
             resValue "string", "flavor_description", "FDroid"
+            dimension "base"
+        }
+
+        withVoip {
+            dimension "voip"
+            resValue "bool", "allow_voip", "true"
+        }
+
+        withoutVoip {
+            dimension "voip"
+            resValue "bool", "allow_voip", "false"
+            packagingOptions {
```

Comment:
This packaging option is not possible at the flavor level. 
The voip support was broken when I tested `appWithVoipDebug` variant. The voip support is restored when I remove this `packagingOptions` from `withoutVoip`.
Please remove it.

Reply:
I don't have this problem. The targets `assembleWithVoipAppfdroidMatrixorg` and `assembleWithoutVoipAppfdroidMatrixorg` are working properly and the `packagingOptions` is well applied in the `withoutVoip` variant.

Reply:
OK, I will try to find a better to remove the useless .so in this case.

Reply:
You mean that you observed the same problem on your side?